File: Python/Mahasiswa.py
# class Decalaration
class Mahasiswa:
    # Membuat Private attributes.
    __nama = ""
    __nim = 0
    __prodi = ""
    __fakultas = ""

    # Constructor with default initialization
    def __init__(self, nama="", nim=0, prodi="", fakultas=""):
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    # Setter and Getter methods for private attributes
    # Set Nama
    def setNama(self, namaMahasiswa):
        self.__nama = namaMahasiswa

    # get Nama
    def getNama(self):
        return self.__nama

    # Set Nim
    def setNim(self, nimMahasiswa):
        self.__nim = nimMahasiswa

    # Get Nim
    def getNim(self):
        return self.__nim

    # Set Prodi
    def setProdi(self, programStudi):
        self.__prodi = programStudi

    # Get Prodi
    def getProdi(self):
        return self.__prodi

    # Set Fakultas
    def setFakultas(self, fakultas):
        self.__fakultas = fakultas

    # Get Fakultas
    def getFakultas(self):
        return self.__fakultas

File: PHP/Mahasiswa.php
<?php
class Mahasiswa
{
    // Membuat atribut Private dari kelas Mahasiswa
    private $nama = '';
    private $nim = '';
    private $prodi = '';
    private $fakultas = '';
    private $image;

    # Constructor with default initialization
    public function __construct($nama = '', $nim = '', $prodi = '', $fakultas = '', $image)
    {
        $this->nama = $nama;
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
        $this->image = $image;
    }

    # Setter and Getter methods for private attributes

    # Set Nama
    public function setNama($namaMahasiswa)
    {
        $this->nama = $namaMahasiswa;
    }

    # Get Nama
    public function getNama()
    {
        return $this->nama;
    }

    # Set Nim
    public function setNim($nimMahasiswa)
    {
        $this->nim = $nimMahasiswa;
    }

    # Get Nim
    public function getNim()
    {
        return $this->nim;
    }

    # Set Prodi
    public function setProdi($programStudi)
    {
        $this->prodi = $programStudi;
    }

    # Get Prodi
    public function getProdi()
    {
        return $this->prodi;
    }

    # Set Fakultas
    public function setFakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }

    # Get Fakultas
    public function getFakultas()
    {
        return $this->fakultas;
    }

    # Set Image
    public function setImage($image)
    {
        $this->image = $image;
    }

    # Get Image
    public function getImage()
    {
        return $this->image;
    }
}
